### PR TITLE
Custom domain

### PR DESCRIPTION
**Pull request by s-soroosh**

### CODE REVIEW

File: fandogh_cli/domain_commands.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    """
+    Domain management commands
+    """
+    pass
+
+
+@click.command("add", cls=FandoghCommand)
+@click.option('--name', prompt='domain name', help='your domain name')
+@login_required
+def add(name):
```
Comment (MahdiZareie) [inline fandogh_cli/domain_commands.py:33]: Isn't it easier if we actually verify inside the `add` command?
We call the API to submit new domain, then ask user to set the TXT record and press enter, and then after user pressed enter we verify the domain, if TXT record was there then we display some message and quite other wise we repeat this process until user Ctrl+C

Comment (MahdiZareie) [reply fandogh_cli/domain_commands.py:33]: 1- POST /api/domains
2- Ask user to set TXT record and press Enter
3-1 if TXT record is there quit
3-2 display a warning and go to 2

Comment (s-soroosh) [reply fandogh_cli/domain_commands.py:33]: Good idea, will do it in this way

Comment (s-soroosh) [reply fandogh_cli/domain_commands.py:33]: done

**Review by MahdiZareie — APPROVED** [review]

**Pull request merged**